gsk4/src/stroke.rs

```diff
@@ -0,0 +1,55 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{LineCap, LineJoin, Stroke};
+
+impl Stroke {
+    // rustdoc-stripper-ignore-next
+    /// Creates a new builder-pattern struct instance to construct a [`Stroke`].
+    ///
+    /// This method returns an instance of [`StrokeBuilder`](crate::builders::StrokeBuilder) which can be used to create a [`Stroke`].
+    pub fn builder(line_width: f32) -> StrokeBuilder {
+        skip_assert_initialized!();
```

Comment:
I'm not sure if this is the correct macro to use

Reply:
It should be `assert_initialized_main_thread`

Reply:
I see, thanks. I'm confused on how it is determined

Reply:
You end up calling gsk::Stroke::new which has `assert_initialized_main_thread`

Reply:
Oh, that makes sense